src/components/TextInput/TextInput.tsx

```diff
@@ -42,6 +44,7 @@ export const TextInput: FunctionComponent<TextInputProps> = ({
 				value={value}
 				disabled={disabled}
 				placeholder={placeholder}
+				{...(ariaLabel ? { 'aria-label': ariaLabel } : {})}
```

Comment:
Wrm spread je die hier als object, mag die `aria-label` geen lege string bevatten?

Reply:
wss wel, maar ik vind geen exacte statement in de spec die het toelaat

Reply:
als ik antwoord krijg op mijn stackoverflow vraag zal ik de code aanpassen:
https://stackoverflow.com/questions/58058203/is-an-html-aria-label-attribute-allowed-to-be-an-empty-string

Reply:
Nee idd, ik vind ook enkel dat het een string moet zijn